client/my-sites/customer-home/cards/features/blogging-prompt/prompts-navigation.jsx

```diff
@@ -67,13 +74,25 @@ const PromptsNavigation = ( { prompts } ) => {
 				} )
 			);
 		}
+
+		// Navigate to the editor.
+		const newPostLink = getNewPostLink();
+		const newPostLinkType = determineUrlType( getNewPostLink() );
+
+		if ( newPostLinkType === URL_TYPE.PATH_ABSOLUTE ) {
+			// Internal link, use router navigation to avoid reloading Calypso.
+			page( newPostLink );
+		} else {
+			// External link, use a full page reload to the new url.
+			window.location.href = newPostLink;
+		}
 	};
```

Comment:
I think that the `navigate( path )` function performs this check https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/navigate/index.ts

Reply:
Thank you, this is the function I was looking for! ❤️